<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthCheck</class>
 <widget class="QWidget" name="AuthCheck">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>权限设置记录</string>
  </property>
  <widget class="QTableView" name="auth_view">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>521</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:solid;
border-width:1px;
border-color:rgb(190,190,190);
background-color: rgb(255, 255, 255);
border-radius: 6px; </string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(78, 160, 201);
font: 11pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>当前角色：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(78, 160, 201);
font: 11pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>数据域选择：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="database">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 5px; 
background-color: rgb(255, 255, 255);
color: rgb(78, 160, 201);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="searchBtn">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>60</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Agency FB&quot;;
color: rgb(255, 255, 255);
background-color: rgb(78, 160, 201);
border-radius: 5px; 
</string>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QLabel" name="rolename_1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(78, 160, 201);
font: 10pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
